tests/sentry/integrations/msteams/test_message_builder.py

```diff
@@ -175,3 +213,143 @@ def test_unlink_indentity_message(self):
         assert 1 == len(unlink_identity_card["body"])
         assert 1 == len(unlink_identity_card["actions"])
         assert "test-url" == unlink_identity_card["actions"][0]["url"]
+
+    def test_issue_message_builder(self):
+        self.event1.data["metadata"].update({"value": "some error"})
```

Comment:
To add description to the issue, type has to be "error" and "metadata.value" will be the description.